### DISCUSSION TITLE
Climate automation based on days and hours

### DISCUSSION THREAD

**Discussion by nickp7** [Blackymas/NSPanel_HA_Blueprint#1411] [General]
Hello, I was wondering if anyone had tried to automate the climate add-on to activate only on specific days and hours. 
I had a look at the esphome climate component but haven't found any mention on this regard...

If someone has a solution/workaround would be much appreciated :)

**Comment by edwardtfn** [comment]
Please take a look at this: https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-Customization#scheduled-climate

**Reply by nickp7** [reply]
Thank you very much :)

**Reply by JS1Dev** [reply]
@nickp7, you might also be interested in the [Scheduler card/custom component](https://community.home-assistant.io/t/scheduler-card-custom-component/217458?page=76)
